docs/sdk/server-side-sdks/python/python-usage.md

```diff
@@ -37,67 +37,86 @@ to be fetched from DevCycle's CDN.
 
     try:
         # Get variable by key for user data
-        variable_value = dvc.variable_value(user, key, 'default-value')
+        variable_value = client.variable_value(user, key, 'default-value')
         print("Variable value is: ", variable_value)
     except Exception as e:
-         print("Exception when calling DevCycleCloudClient->variable_value: %s\n" % e)
+         print(f"Exception when calling DevCycleLocalClient->variable_value: {e}")
 
 ```
 
-The default value can be of type string, boolean, number, or object.
+The default value can be of type string, boolean, number, or dictionary.
 
 If you would like to get the full Variable you can use `variable()` instead. This contains fields such as:
 `key`, `value`, `type`, `defaultValue`, `isDefaulted`.
 
 ## Getting All Variables
 
-To get all variables, use the `all_variables()` method to retrieve a dict with all variables that the user receives.
+Use the `all_variables()` method to retrieve a dictionary with all the segmented variables for the user.
 
 ```python
     try:
```

Comment:
Should all these examples be indented? It looks a little bit off on the page: 
<img width="934" alt="image" src="https://github.com/DevCycleHQ/devcycle-docs/assets/1219069/20b31971-1bd3-49b4-8899-67ff145a0c48">

Reply:
agreed. i'll remove the indent.  I think it was there already and I just didn't think to fix it.